### DISCUSSION TITLE
Plausible vs Google Analytics: differences in most viewed pages (numbers and time)

### DISCUSSION THREAD

**Discussion by ingejanse** [plausible/analytics#1727] [Anything Else]
I just made the switch from GA to Plausible (thank you), and I'm now comparing numbers (februari 2022) to report properly to my team about what this switch means for our statistics. Some numbers are quite identical, but some do really differ a lot. What stands out most is the difference in viewed pages and time on page. The list for GA is quite different than for Plausible, and the time on site is almost absurd long. Can someone explain this, so that I can account for this in my team? Thanks!

![Schermafbeelding 2022-03-01 om 09 34 35](https://user-images.githubusercontent.com/31367673/156133978-b2a73248-1b94-4148-87d3-f612752bc6e0.png)

![Schermafbeelding 2022-03-01 om 09 34 26](https://user-images.githubusercontent.com/31367673/156133984-5eb389c8-aa32-49e1-98fe-98e3975b3b00.png)

**Comment by metmarkosaric** [comment]
hi @ingejanse! and thanks for the kind words! 

pageviews looks pretty similar. analytics tools work differently so you can rarely expect identical results no matter how many different providers you test. we recommend you try out [our proxy](https://plausible.io/docs/proxy/introduction) too in case you'd like to count visitors that may use privacy-friendly browsers or adblockers extensions. 

for time on page, the difference comes from the different way we define the metric. in GA bounced visitors are included in the time on page metric but all set to 10 seconds time on page while in Plausible we exclude bounced visitors from the time on page calculation. Neither of these approaches are 100% accurate (Plausible would show higher than the actual time on page, GA would show lower than the actual time on page) as in both cases it's not possible to measure the time on page of people that bounce from your site. 

this is why we're working on a new approach with the [beacon API](https://github.com/plausible/analytics/discussions/190) which would allow us to include actual time on page even from people that bounce and which would make Plausible have more accurate time on page than what's possible now in GA and in Plausible. it's a big project and we just started testing the first version last week so if everything works well, we're hoping to introduce it at some point this year.

i hope this helps, thanks!
